src/test/java/org/openmrs/contrib/qaframework/automation/RolesAndPrivilegesSteps.java

```diff
@@ -117,6 +117,8 @@ public void editPrivilege(){
     public void editRole(){
     	administrationManageRolesPage.goToEditRole();
     	administrationManageRolesPage = addRolePage.addDescription("Developers of the OpenMRS...edited for the e2e automation test");
+    	administrationManageRolesPage = addRolePage.selectPrivileges();
```

Comment:
i would have renamed this class as manageRolesPage instead of AdministrationManageRolesPage

Reply:
There's already a page named that